CMakeLists.txt

```
@@ -125,20 +125,43 @@ include_optional_module(ENVIRONMENT_VARIABLE WITH_VARIABLE_SPLASH_SCREEN
                         OPTION_VARIABLE USE_VARIABLE_SPLASH_SCREEN
                         READABLE_NAME "build-type splash screen")
 include_optional_module(ENVIRONMENT_VARIABLE WITH_OWN_QTKEYCHAIN
-                        OPTION_VARIABLE USE_OWN_QT5KEYCHAIN
-                        READABLE_NAME "own QKeychain (Qt5) library")
+                        OPTION_VARIABLE USE_OWN_QTKEYCHAIN
+                        READABLE_NAME "own QtKeychain library")
 include_optional_module(ENVIRONMENT_VARIABLE WITH_MAIN_BUILD_SYSTEM
                         OPTION_VARIABLE USE_MAIN_BUILD_SYSTEM
                         READABLE_NAME "original CI (setup-windows-sdk.ps1) windows build system"
                         SUPPORTED_SYSTEMS "Windows")
+include_optional_module(ENVIRONMENT_VARIABLE WITH_QT6
+                        OPTION_VARIABLE USE_QT6
+                        READABLE_NAME "build with Qt6 (if installed)")
+
+if(USE_QT6)
+  find_package(
+          Qt6 6.2.0
+          QUIET)
+  if(NOT Qt6_FOUND)
+    message(STATUS "Qt6 >= 6.2.0 was not found. Attempting to build with Qt5.")
```

```suggestion
    message(STATUS "Qt6 >= 6.2.0 was not found. Building with Qt5.")
```

Everything is an attempt really, but this sounds better when you're compiling and looking at this message hundreds of times. A bit better developer UX.